tests/test_cases/test_vhdl_access/test_vhdl_access.py

```diff
@@ -91,3 +91,22 @@ def check_instance(obj, objtype):
 
     if fails:
         raise TestFailure("%d Failures during the test" % fails)
+
+@cocotb.test()
+def port_not_hierarchy(dut):
```

Comment:
@stuarthodgson this test causes Aldec to segfault.

Looking further it's in the `check_and_store` duplicate detection when we `delete hdl` if a duplicate is detected.  Commenting out the `delete` makes the test pass....